test/postgres.test.coffee

```diff
@@ -66,6 +71,11 @@ test['Postgres flavour'] =
       toString: ->
         assert.same @upd.toString(), 'UPDATE table SET field = 1 FROM table2'
 
+    '>> table(table).set(field, 1).with(alias, table)':
+      beforeEach: -> @upd.table('table').set('field', 1).with('alias', squel.select().from('table'))
+      toString: ->
```

Comment:
Please add a `toParam()` test too, and the `with` query should also use params, e.g. `select().from('table').where('a = ?', 5)`